src/Workleap.Authentication.ClientCredentialsGrant.Tests/OpenAPI/OpenApiSecurityDescriptionTests.cs

```diff
@@ -20,7 +21,11 @@ public async Task Given_API_With_Client_Credentials_Attribute_When_Generating_Op
             .WithWorkingDirectory(projectFolder)
             .WithValidation(CommandResultValidation.None)
             .WithArguments(["build", "--no-incremental"])
-            .ExecuteAsync();
+            .ExecuteBufferedAsync();
+
+        testOutputHelper.WriteLine("Build output:");
+        testOutputHelper.WriteLine(result.StandardError);
+        testOutputHelper.WriteLine(result.StandardOutput);
```

Comment:
Helped me understand why the compilation failed when I didn't have the proper .NET SDK based on global.json.